env0/data_oidc_credentials_test.go

```diff
@@ -0,0 +1,128 @@
+package env0
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/env0/terraform-provider-env0/client/http"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestOidcCredentialDataSource(t *testing.T) {
+	tests := [][]string{
```

Comment:
runs tests for all 3 types by iterating.

Reply:
nice